Command/CronStartCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * This file is part of the SymfonyCronBundle package.
+ *
+ * (c) Dries De Peuter <dries@nousefreak.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Cron\CronBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Alexander Lokhman <alex.lokhman@gmail.com>
+ */
+class CronStartCommand extends ContainerAwareCommand
+{
+    const PID_FILE = '.cron-pid';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('cron:start')
+            ->setDescription('Starts cron scheduler')
+            ->addOption('blocking', 'b', InputOption::VALUE_NONE, 'Run in blocking mode.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if ($input->getOption('blocking')) {
+            $output->writeln(sprintf('<info>%s</info>', 'Starting cron scheduler in blocking mode.'));
+            $this->scheduler($output->isVerbose() ? $output : new NullOutput(), null);
+
+            return 0;
+        }
+
+        if (!extension_loaded('pcntl')) {
+            throw new \RuntimeException('This command needs the pcntl extension to run.');
+        }
+
+        $pidFile = getcwd().'/'.self::PID_FILE;
```

Comment:
I would suggest not writing the PID file in the working directory. Instead place it in a tmp or data folder.

Reply:
Yeah, I initially thought the same but decided to use the same strategy, as `symfony/web-server` component uses for it's PID file. However, I agree that temp folder is the better solution (`/var/run` would be ideal but temp should be enough) for now. I will update it.